tests/integration/extension/src/background.js

```diff
@@ -1,15 +1,52 @@
 /* eslint-disable no-unused-vars */
 import * as webScience from "@mozilla/web-science"
+import browser from "webextension-polyfill";
```

Comment:
What is super intriguing for me is that despite including polyfill here, I had to still include it as the first script in manifest's background section. Somewhere during bundling rollup's tree shaking is stripping things we need at the runtime in browser.

Reply:
Hm, I've managed to get that working... glean.js doesn't import it though and requires the polyfill at runtime :/ so I haven't been worrying about it too much. I do like having the import so I can get richer types in my IDE though.